src/components/ConversationSettings/ConversationPermissionsSettings.vue

```diff
@@ -221,6 +221,7 @@ export default {
 		getPermissionRadioValue(value) {
 			switch (value) {
 			case PERMISSIONS.MAX_DEFAULT:
+			case PERMISSIONS.MAX_CUSTOM:
```

Comment:
Could we do the same for 'Restricted'?

https://user-images.githubusercontent.com/93392545/223188099-83d032c0-cbe7-4d78-bd2d-762e4efdea0f.mp4

Reply:
Added